tests/approaches/test_oracle_approach.py

```diff
@@ -347,6 +348,32 @@ def test_oracle_approach_cover_multistep_options():
         assert policy_solves_task(policy, task, env.simulate)
 
 
+@longrun
+def test_longrun_oracle_approach_cover_multistep_options():
```

Comment:
wow so this takes >100 seconds? even for a longrun test, this is quite slow

Reply:
reduced to 20 seconds